app/controllers/users_controller.rb

```diff
@@ -0,0 +1,87 @@
+class UsersController < ApplicationController
+  include ProviderHelper
+  before_action :set_user, only: [:show, :update, :destroy, :questions, :tags]
+
+  def index
+    @users = User.paginate(page: params[:page])
+    render json: @users, status: :ok
+
+    # add meta data for pagination
+    # total: resource.total_entries,
+    # total_pages: resource.total_pages,
+    # first_page: resource.current_page == 1,
+    # last_page: resource.next_page.blank?,
+    # previous_page: resource.previous_page,
+    # next_page: resource.next_page,
+    # out_of_bounds: resource.out_of_bounds?,
+    # offset: resource.offset
```

Comment:
:thought_balloon: left it as a comment so it could be referenced later. Would normally put these comments in a `.gitignored` file but thought others might need this as well